### ISSUE TITLE
Revise pages that have 2 columns

### ISSUE THREAD

**Issue by cthomasONRR** [ONRR/onrr.gov-site#1691]
Webpages that either need the right rail (8 and 4 size columns) adjusted or right rail cards 'shrunk'

- [x] https://dev-onrr-frontend.app.cloud.gov/about/interagency?tab=outer-continental-shelf-mou (map could be moved more to the right)
- [x] https://dev-onrr-frontend.app.cloud.gov/about/foia (contact card shrink)
- [x] https://dev-onrr-frontend.app.cloud.gov/indian/assistance?tab=federal-indian-minerals-office (card shrink)
- [x] https://dev-onrr-frontend.app.cloud.gov/compliance-enforcement/cooperative?tab=ipa-fellowship-program (card shrink)
- [x] https://dev-onrr-frontend.app.cloud.gov/paying/interest (card shrink)
- [x] https://dev-onrr-frontend.app.cloud.gov/paying/unpaid-debt (card shrink)
- [x] https://dev-onrr-frontend.app.cloud.gov/references/handbooks/indian-payor-handbook?tab=chapter-2-indian-valuation-basics (card shrink)
- [x] https://dev-onrr-frontend.app.cloud.gov/references/handbooks/indian-payor-handbook?tab=chapter-3-indian-oil-valuation (card shrink)
- [x] https://dev-onrr-frontend.app.cloud.gov/references/valuation/processed-gas-example?tab=detailed-example (card shrink)
- [x] https://dev-onrr-frontend.app.cloud.gov/references/valuation/pop-example?tab=overview (card shrink)
- [x] https://dev-onrr-frontend.app.cloud.gov/references/valuation/pop-example?tab=detailed-example (card shrink)
- [x] https://dev-onrr-frontend.app.cloud.gov/compliance-enforcement/compliance (adjust image alignment)

**Comment by mcharg** [comment]
I updated these pages:
- [ ] https://dev-onrr-frontend.app.cloud.gov/indian/assistance?tab=indian-energy-service-center

**Comment by cthomasONRR** [comment]
PDF icon mysteriously appeared in System Access title. I was not editing that content block, and was also not doing anything with PDFs...
https://dev-onrr-frontend.app.cloud.gov/getting-started/system-access
![image](https://user-images.githubusercontent.com/72526208/157314519-9c10a6ab-26c8-44a3-a9d9-dd8cbcc78508.png)

**Comment by cthomasONRR** [comment]
@mcharg I've fixed all the ones listed above. Let me know if you find anymore. I cruised through the site, couldn't find any other cards that needed adjusting.

**Comment by mcharg** [comment]
@cthomasONRR These all look good.

**Comment by cthomasONRR** [comment]
@mcharg 

I added the 8 and 4 layout column blocks to the foia page, just to make sure I understand how to use them. I can delete them out if you think they're unnecessary on that page. 

You had already added them to the only 2 pages I identified as needing them (FIMO and IPA Fellowship Program tabs). None of the other pages have multiple cards on the right rail, so adding these layout column blocks seems unnecessary right now. 

I can't use this layout for the examples or Indian payor handbook, because the right-rail cards need to be associated with specific content.

I think we're good to close this issue.
